lib/openshift/project_resource.rb

```diff
@@ -112,13 +112,13 @@ def self.bulk_update(user:, resources:, quiet: true)
     end
 
     # @param grace_period [Boolean] useful to add the pod delete parameter
-    def delete(by: nil, grace_period: nil)
+    def delete(by: nil, grace_period: nil, wait: false)
       by = default_user(by)
       del_opts = {}
       del_opts[:grace_period] = grace_period unless grace_period.nil?
       by.cli_exec(:delete, object_type: self.class::RESOURCE,
                object_name_or_id: name, namespace: project.name,
-               **del_opts)
+               wait: "#{wait}", **del_opts)
```

Comment:
was going to comment on the incorrect format yesterday, but looks like you updated it already. :)